rclcpp/include/rclcpp/publisher_base.hpp

```diff
@@ -139,6 +139,12 @@ class PublisherBase : public std::enable_shared_from_this<PublisherBase>
   size_t
   get_intra_process_subscription_count() const;
 
+  /// Get if durability is transient local
+  /** \return If durability is transient local*/
+  RCLCPP_PUBLIC
+  bool
+  is_durability_transient_local() const;
```

Comment:
do we need this method and the `durability_is_transient_local_` ? the data can already be queried and if we add this, we should also add short notations for all of them

Reply:
I removed durability_is_transient_local_ since qos can be retrieved with publisher_handle_ in publisher_base.